spec/forms/pages/type_of_answer_form_spec.rb

```diff
@@ -38,29 +37,37 @@
   end
 
   describe "#submit" do
-    let(:session_mock) { {} }
-
     it "returns false if the form is invalid" do
       allow(type_of_answer_form).to receive(:invalid?).and_return(true)
-      expect(type_of_answer_form.submit(session_mock)).to be_falsey
-    end
-
-    it "sets a session key called 'page' as a hash with the answer type in it" do
-      type_of_answer_form.answer_type = "email"
-      type_of_answer_form.submit(session_mock)
-      expect(session_mock[:page]).to include(answer_type: "email")
+      expect(type_of_answer_form.submit).to be_falsey
```

Comment:
I know it's in the original code here, but wondering if you know why we're using `be_falsey` here instead of `be false`?

Reply:
Not essential to this PR but if `be_falsey` isn't needed then we could potentially make the expectation more specific

Reply:
I'm going to leave this as is for the moment, but we do need to review our specs more generally and refactor those too.